src/da-layer/services/EmotionsService.ts

```diff
@@ -0,0 +1,15 @@
+import { IEmotionsService } from '../abstract/IEmotionsService';
+import { endpoint } from '../apiEndpoint';
+import { Emotions } from '../models/Emotions';
+
+export class EmotionsService implements IEmotionsService {
+  public getEmotions(): Promise<Emotions> {
+    return new Promise((resolve, reject) => {
+      fetch(endpoint)
+        .then(res => {
+          resolve((res as unknown) as Emotions);
```

Comment:
Fetch ресолвит Response, который никак не похож на Emotions
```suggestion
          res.json().then(data => resolve(data as Emotions))
```

Reply:
Поправил